Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
A new cpp file needs to be added to `Gems\PhysX\Code\physx_editor_tests_files.cmake` so it gets compiled.

Reply:
Added to `Gems\PhysX\Code\physx_tests_files.cmake` instead since these tests were not scoped for the editor tests.Z

However, this allowed me to discover the existing editor tests around related areas and I tossed in a couple simple tests with this PR and added that test file to `Gems\PhysX\Code\physx_editor_tests_files.cmake`.

Reply:
Oh yeah, I advised the wrong file :) sorry about that. But it's good you spotted other errors :)